GETTING_STARTED.md

```diff
@@ -0,0 +1,63 @@
+# Getting Started as a Boutiques Developer
+
+This guide will help you set up your local environment for coding and testing Boutiques.
+
+## 0. Things you'll need
+- **A Linux system**. It _may_ be possible to develop Boutiques on Windows, but will likely cause many headaches.

```

Comment:
Maybe update this to say a **Linux shell**, since Mac works. Also potentially add a parenthetical at the end of the sentence that is "(if you successfully develop for Boutiques on Windows, please update this guide with details!)"